server/handle_sync_get_blob.go

```diff
@@ -47,15 +53,66 @@ func (s *Server) handleSyncGetBlob(e echo.Context) error {
 
 	buf := new(bytes.Buffer)
 
-	var parts []models.BlobPart
-	if err := s.db.Raw("SELECT * FROM blob_parts WHERE blob_id = ? ORDER BY idx", nil, blob.ID).Scan(&parts).Error; err != nil {
-		s.logger.Error("error getting blob parts", "error", err)
-		return helpers.ServerError(e, nil)
-	}
+	if blob.Storage == "sqlite" {
```

Comment:
No change to the logic, of retrieving the blob parts, just wrapped in an if statement